dozer-sql/src/pipeline/aggregation/avg.rs

```diff
@@ -26,6 +26,7 @@ impl AvgAggregator {
             FieldType::Decimal => FieldType::Decimal,
             FieldType::Float => FieldType::Float,
             FieldType::Int => FieldType::Int,
+            FieldType::UInt => FieldType::UInt,
```

Comment:
In the case of AVG the return type can be either an integer or a float.
For Int and UInt, we can return Float by default to cover both cases

Reply:
decimal -> decimal
i64 -> float
u64 -> float
float -> float